concent_api/core/message_handlers.py

```diff
@@ -331,7 +336,11 @@ def handle_send_force_subtask_results(request, client_message: message.concents.
             reason = message.concents.ServiceRefused.REASON.DuplicateRequest,
         )
 
-    if not base.is_provider_account_status_positive(request):
+    if not base.is_account_status_positive(  # pylint: disable=no-value-for-parameter
+        concent_rpc             = base.get_concent_rpc_instance(),  # pylint: disable=no-value-for-parameter
+        requestor_eth_address   = client_message.ack_report_computed_task.report_computed_task.task_to_compute.requestor_ethereum_address,
```

Comment:
`task_to_compute` is available now directly as a variable, please use it.

Reply:
`task_to_compute` restored and repaird